src/main/java/com/notably/logic/suggestion/commands/OpenSuggestionCommand.java

```diff
@@ -0,0 +1,47 @@
+package com.notably.logic.suggestion.commands;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.notably.commons.core.path.Path;
+import com.notably.logic.suggestion.stubs.SuggestionListStub;
+import com.notably.model.Model;
+import com.notably.model.suggestion.SuggestionItem;
+import com.notably.model.suggestion.SuggestionItemImpl;
+
+/**
+ * Represents a suggestion command object to open a note.
+ */
+public class OpenSuggestionCommand implements SuggestionCommand {
+    private Path path;
```

Comment:
We're deprecating `Path`. @ljiazh3ng is working on a new set of `AbsolutePath` and `RelativePath` in #116 . To be merged soon!

Please keep a watch for it :)